src/umpire/ResourceManager.cpp

```diff
@@ -137,6 +139,16 @@ ResourceManager::ResourceManager() :
 ResourceManager::~ResourceManager()
 {
   for (auto&& allocator : m_allocators) {
+    if (allocator->getCurrentSize() != 0) {
+      std::stringstream ss;
+
+      umpire::print_allocator_records(Allocator{allocator.get()}, ss);
+
+      UMPIRE_LOG(Error, allocator->getName() << " Allocator still has "
```

Comment:
Error means it will be printed to stdout by default. Do we want to add a variable to control where it gets printed?

Reply:
I think this behaviour changed at some point.  I am not seeing this come to stdout.  I only see it in the log file.